### PR TITLE
e2e: allow variable tx size 

### PR DESCRIPTION
**Pull request by cmwaters**
Closes: #5905

This also simplifies queue type to a testnet wide config and fixes a bug with using the `hybrid` p2p testnet setup

### CODE REVIEW

File: test/e2e/generator/generate.go [hunk 1]
```diff
@@ -189,7 +189,7 @@ func generateTestnet(r *rand.Rand, opt map[string]interface{}) (e2e.Manifest, er
 			nextStartAt += 5
 		}
 		node := generateNode(r, e2e.ModeFull, startAt, manifest.InitialHeight, false)
-		node.QueueType = manifest.QueueType
```
Comment (williambanfield) [inline test/e2e/generator/generate.go:192]: How is this line related to adding different Tx sizes?

Comment (cmwaters) [reply test/e2e/generator/generate.go:192]: It's a minor change that I thought doesn't warrant it's own PR. I just commented in the description:
> This also simplifies queue type to a testnet wide config (instead of a per node config)

**Comment by tychoish** [comment]
I'd leave the per-node queue type configuration, but the rest of this seems legit.

**Review by tychoish — APPROVED** [review]

**Review by tychoish — APPROVED** [review]
